### PR TITLE
Pixelmon review

### PR DESCRIPTION
**Pull request by lauchness**
## Description
This PR updates the pixelmon config

It's built on top of #296 and includes the custom `interactWithNFTLink` work there for pixelmon

Screen Shot 2023-08-16 at 7 08 12 PM

### CODE REVIEW

File: src/config/partners/pixelmon.ts [hunk 1]
```diff
@@ -17,17 +17,36 @@ const pixelmon: Partner = {
...
         '/partners/pixelmon/drops/FINAL_ART-Kevin-Nanoblock-Illustration-NFT-Manfred Ho.jpg',
-      creator: '0xCe0d48718F195aa0Ee14121ED40AE52F0EA55159',
+      creator: '0xF0A72A215636930c369e220c9fF69861E721103A',
       name: 'Summer Kevin',
-      externalLink: 'https://www.pixelmon.ai/kevin-nanoblock',
+      type: 'erc-721',
+      startDate: Date.UTC(2023, 7, 17, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
+      endDate: Date.UTC(2023, 8, 1, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
+      price: '0.001',
+      description: 'Fun Fact: Do you know that Kevin cannot swim?',
```
Comment (AshishYelekar) [inline src/config/partners/pixelmon.ts:26]: Minor thing - this should be "Did you know" instead of "Do you know"

**Review by AshishYelekar — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for AshishYelekar @ 2023-08-16 23:21:39 UTC
User cannot review their own commit

File: src/config/partners/pixelmon.ts (hunk 1 above)
Comment (AshishYelekar) [reply src/config/partners/pixelmon.ts]: Nvm, I updated it!

**Review by wilsoncusack — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-17 00:11:38 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**